qiita_ware/test/test_processing_pipeline.py

```diff
@@ -257,20 +324,9 @@ def test_get_preprocess_fasta_cmd_sff_no_run_prefix(self):
         self.assertEqual(obs_cmds[3], exp_cmd_4)
 
     def test_get_preprocess_fasta_cmd_sff_run_prefix(self):
-        # Need to alter the run_prefix of one sample so we can test the
-        # multiple values
-        conn_handler = SQLConnectionHandler()
-        sql = ("UPDATE qiita.prep_1 SET run_prefix='test1' WHERE "
-               "sample_id = '1.SKM9.640192'")
-        conn_handler.execute(sql)
-
-        raw_data = RawData(3)
         params = Preprocessed454Params(1)
```

Comment:
the id of the first Peprocessed454Params, I just need one instance of them to be able to call the preprocessing pipeline